datafusion/core/src/physical_plan/aggregates/no_grouping.rs

```diff
@@ -164,7 +164,7 @@ impl RecordBatchStream for AggregateStream {
 
 /// Perform group-by aggregation for the given [`RecordBatch`].
 ///
-/// If successfull, this returns the additional number of bytes that were allocated during this process.
+/// If successfully, this returns the additional number of bytes that were allocated during this process.
```

Comment:
```suggestion
/// If successful, this returns the additional number of bytes that were allocated during this process.
```

(very minor, in case you have to update the PR again )

Reply:
I also find a same new typo, look like I should update again 🤣